lib/hdpublickey.js

```diff
@@ -65,6 +65,22 @@ function HDPublicKey(arg) {
   }
 }
 
+
+/**
+ * Verifies that a given path is valid.
+ *
+ * @param {string|number} arg
+ * @return {boolean}
+ */
+HDPublicKey.prototype.isValidPath = function(arg) {
```

Comment:
I wonder if instead of a try/catch here we could use a regex to validate the path?